packages/react-components/src/components/RTT.tsx

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/* @conditional-compile-remove(rtt) */
+import { IPersona, Persona, Stack, PersonaSize, Text, useTheme } from '@fluentui/react';
+/* @conditional-compile-remove(rtt) */
+import React from 'react';
+/* @conditional-compile-remove(rtt) */
+import { OnRenderAvatarCallback } from '../types';
+/* @conditional-compile-remove(rtt) */
+import {
+  captionClassName,
+  captionsContentContainerClassName,
+  displayNameClassName,
+  displayNameContainerClassName,
+  iconClassName,
+  isTypingClassName,
+  rttContainerClassName
+} from './styles/Captions.style';
+/* @conditional-compile-remove(rtt) */
+import { _CaptionsInfo } from './CaptionsBanner';
+/* @conditional-compile-remove(rtt) */
+import { useLocale } from '../localization';
+
+/* @conditional-compile-remove(rtt) */
+/**
+ * @beta
+ * strings for rtt
+ */
+export interface RTTStrings {
+  /**
+   * String indicating that the user is typing
+   */
+  isTypingText?: string;
+}
+
+/* @conditional-compile-remove(rtt) */
+/**
+ * @beta
+ * Props for a single line of RTT.
+ */
+export interface RTTProps {
+  /**
+   * RTT id
+   */
+  id: string;
+  /**
+   * Display name of the user
+   */
+  displayName: string;
+  /**
+   * RTT content
+   */
+  captionText: string;
+  /**
+   * user id of the user
+   */
+  userId?: string;
+  /**
+   * Optional callback to override render of the avatar.
+   *
+   * @param userId - user Id
+   */
+  onRenderAvatar?: OnRenderAvatarCallback;
+  /**
+   * Boolean indicating whether the RTT is still in progress
+   */
+  isTyping?: boolean;
+  /**
+   * Strings for RTT
+   */
+  strings?: RTTStrings;
+}
+
+/* @conditional-compile-remove(rtt) */
+/**
+ * @beta
+ * A component for displaying a single line of RTT
+ */
+export const RTT = (props: RTTProps): JSX.Element => {
```

Comment:
RealTimeTextCaption? I feel this is more readable than RTT itself, wondering what is the naming for teams

Reply:
Or without caption, I feel CTE/RTT is kinda thing could be changed in our doc in the future, and people will be totally lost when they read the name at that time